container/raw/handler.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/google/cadvisor/fs"
 	info "github.com/google/cadvisor/info/v1"
 	"github.com/google/cadvisor/machine"
+	"github.com/opencontainers/runc/libcontainer/cgroups"
```

Comment:
Should this import be in a different group?

Reply:
Same thing; before this PR the imports looked like this:

```go
import (
        "fmt"
        "io/ioutil"
        "path"
        "strconv"
        "strings"
        "time"

        "github.com/google/cadvisor/container"
        "github.com/google/cadvisor/container/common"
        dockerutil "github.com/google/cadvisor/container/docker/utils"
        containerlibcontainer "github.com/google/cadvisor/container/libcontainer"
        "github.com/google/cadvisor/devicemapper"
        "github.com/google/cadvisor/fs"
        info "github.com/google/cadvisor/info/v1"
        "github.com/google/cadvisor/zfs"

        dockercontainer "github.com/docker/docker/api/types/container"
        docker "github.com/docker/docker/client"
        "golang.org/x/net/context"
        "k8s.io/klog/v2"
)
```

The split between the last two groups seems random, and I am not sure to which group should I add `"github.com/opencontainers/runc/libcontainer/cgroups"`.